src/logic/map_objects/tribes/worker_program.cc

```diff
@@ -551,6 +551,8 @@ walk
       * ``coords``: Walk to a previously found and selected field/coordinate.
       * ``object-or-coords``: Walk to a previously found and selected object if present;
         otherwise to previously found and selected field/coordinate.
+      * ``ne|e|se|sw|w|nw``: New in version 1.2. Orders the worker to move
+        one field in the specified direction. Fails if the movement is not possible.
```

Comment:
Maybe this would be more clear?

```suggestion
        one field in the specified direction. The worker program exits with "fail" if the movement is not possible.
```